sources/packages/web/src/views/institution/student/applicationDetails/StudentAppealRequest.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <student-appeal-requests-approval
```

Comment:
Asking for clarification, did we decide that institution can see the staff area in the appeal request forms?

![image](https://github.com/bcgov/SIMS/assets/54600590/11a75ee2-9a17-47a3-b536-5397ee2ce35d)

Reply:
![image](https://github.com/bcgov/SIMS/assets/77353155/b23dcb32-0264-44e9-97b0-51d2c2b1ae6f)

Reply:
Ah, Sorry I missed it